lib/ansible/modules/cloud/amazon/ec2_vpc_vpn_info.py

```diff
@@ -120,26 +120,26 @@
              returned: always
              type: int
              sample: 0
-         last_status_change:
-             description: The date and time of the last change in status.
-             returned: always
-             type: datetime
-             sample: 2018-02-09T14:35:27+00:00
-         outside_ip_address:
-             description: The Internet-routable IP address of the virtual private gateway's outside interface.
-             returned: always
-             type: str
-             sample: 13.127.79.191
-         status:
-             description: The status of the VPN tunnel.
-             returned: always
-             type: str
-             sample: DOWN
-         status_message:
-             description: If an error occurs, a description of the error.
-             returned: always
-             type: str
-             sample: IPSEC IS DOWN
+           last_status_change:
```

Comment:
Wow, thanks for catching that. There was recently a [new one](https://aws.amazon.com/about-aws/whats-new/2019/08/aws-site-to-site-vpn-now-supports-certificate-authentication/) added that may be returned too:
```
certificate_arn:
    description: The Amazon Resource Name of the virtual private gateway tunnel endpoint certificate.
    returned: when a private certificate is used for authentication
    type: str
    sample: "arn:aws:acm:us-east-1:123456789101:certificate/c544d8ce-20b8-4fff-98b0-example"
```

Reply:
As discussed on IRC, this will be added in a follow-up PR.

Reply:
I've added it to #63541.